core/src/main/scala/org/apache/spark/scheduler/TaskSchedulerImpl.scala

```diff
@@ -704,12 +704,12 @@ private[spark] object TaskSchedulerImpl {
    * Used to balance containers across hosts.
    *
    * Accepts a map of hosts to resource offers for that host, and returns a prioritized list of
-   * resource offers representing the order in which the offers should be used.  The resource
+   * resource offers representing the order in which the offers should be used. The resource
    * offers are ordered such that we'll allocate one container on each host before allocating a
    * second container on any host, and so on, in order to reduce the damage if a host fails.
    *
-   * For example, given <h1, [o1, o2, o3]>, <h2, [o4]>, <h1, [o5, o6]>, returns
-   * [o1, o5, o4, 02, o6, o3]
+   * For example, given {@literal <h1, [o1, o2, o3]>}, {@literal <h2, [o4]>} and
+   * {@literal <h3, [o5, o6]>}, returns {@literal [o1, o5, o4, o2, o6, o3]}.
```

Comment:
It seems we can't use `@code` here if there are codes such as `<A ...>` (it seems `< A...>` case looks fine. I ran some tests with the comments below:

```
 * For example, given {@code < h1, [o1, o2, o3] >}, {@code < h2, [o4]>} and {@code <h3, [o5, o6]>},
 * returns {@code [o1, o5, o4, o2, o6, o3]}.
 *
 * For example, given
 *
 * {@code <h1, [o1, o2, o3]>},
 *
 * {@code <h2, [o4]} and
 *
 * {@code h3, [o5, o6]>},
 *
 * returns {@code [o1, o5, o4, o2, o6, o3]}.
```

**Scaladoc**

![2017-04-12 4 34 04](https://cloud.githubusercontent.com/assets/6477701/24947422/b02fc452-1fa1-11e7-90cb-55079edf6acb.png)


**Javadoc**

![2017-04-12 4 34 38](https://cloud.githubusercontent.com/assets/6477701/24947418/a9235f3e-1fa1-11e7-8aab-d7c41279a67a.png)

If we use `@literal`, it seems fine.

**Scaladoc**

![2017-04-12 4 46 54](https://cloud.githubusercontent.com/assets/6477701/24947467/e6ded6fa-1fa1-11e7-9cd7-a24c0b3778da.png)


**Javadoc**

![2017-04-12 4 46 43](https://cloud.githubusercontent.com/assets/6477701/24947470/e95768fc-1fa1-11e7-8bd4-1983208a87a6.png)

This seems not exposed in the API documentation anyway.